sherpa/astro/io/crates_backend.py

```diff
@@ -242,38 +262,25 @@ def _require_key(crate, name, dtype=str):
     return key
 
 
-def _try_col(crate, colname, make_copy=False, fix_type=False,
-             dtype=SherpaFloat):
-    """
-    checked for new crates
-    """
+def _try_col(crate: TABLECrate,
+             colname: str,
+             make_copy: bool = False,
+             fix_type: bool = False,
+             dtype: type = SherpaFloat) -> Optional[np.ndarray]:
+    """Return the column data or return None."""
```

Comment:
I was just suggesting to add "is the column does not exist" to the docstring, but GH seemed to have changed their website and I could not make "suggestions" when I reviewed this so I wrote it in plain text. Sorry that was not clear. (Now the button is there again.)

I agree that it would be *nice* to unify the backend code and remove or combine some of those utility functions, but is that the best use of our time?